Sources/TarkaUI/Components/Input Field/Model/TUIInputFieldItem.swift

```diff
@@ -28,11 +31,17 @@ public struct TUIInputFieldItem: Equatable, Hashable {
   ///   - value: A string that holds the value ie. content description
   public init(
     style: TUIInputFieldStyle,
-    title: String = "", value: String = "") {
+    title: String = "", value: String = "",
+    highlightBarColor: Color = .primaryTUI,
+    isHighlightBarEnabled: Bool = false,
+    stateType: TUIInputFieldState = .none ) {
```

Comment:
create view modifiers

Reply:
I feel Its better if we make the entire styling and inputs to the TextField with this InputFieldItem itself.